pages/api/plans/find/index.js

```diff
@@ -0,0 +1,25 @@
+import { findPlans } from '../../../../lib/dependencies';
+import { ArgumentError } from '../../../../lib/domain';
+
+export const endpoint = ({ findPlans }) => async (req, res) => {
+  try {
+    const result = await findPlans({
+      firstName: req.body.firstName,
+      lastName: req.body.lastName,
+      systemIds: req.body.systemIds
+    });
+
+    res.status(200).json(result);
+  } catch (err) {
+    //log error here
```

Comment:
This gets automatically logged (assuming it is from the use case). The use cases are wrapped in a `withLogger` function that logs request, response and errors.

Reply:
ok great, we can remove the "//log error here" then

Reply:
...actually we're going to add some anyway. So that if this endpoint gets more complicated we don't forget to add logging - more logging is better than not enough logging 😅

Reply:
haha nice